i18n.md

```diff
@@ -0,0 +1,54 @@
+# Electron Internationalization
+
+🌎 🌍 🌏
```

Comment:
This is the most random comment to make, but I think if you technically wanted the order to reflect the Earth rotating, it should be the other way around. 😛

🌏 🌍 🌎